tests/php/Model/SiteTreeTest.php

```diff
@@ -1380,10 +1384,23 @@ public function testClassDropdown()
 
         $this->assertArrayNotHasKey(SiteTreeTest_NotRoot::class, $method->invoke($rootPage));
         $this->assertArrayHasKey(SiteTreeTest_NotRoot::class, $method->invoke($nonRootPage));
+        foreach ([SiteTreeTest_ClassA::class, SiteTreeTest_ClassB::class] as $className) {
+            $otherPage = new $className();
+            $otherPage->write();
+            $result = $method->invoke(object: $otherPage);
+            $this->assertEquals(array_key_first($result), $className);
+            // remove the first element as this is not alphabetical
+            array_shift($result);
+            // create a sorted array
+            $resultSorted = $result;
+            asort($resultSorted);
+            $this->assertEquals(implode('_', $result), implode('_', $resultSorted));
```

Comment:
You don't need to stringify the arrays, assertEquals would compare them, including the order of elements.